runtime/src/lib.rs

```diff
@@ -1622,7 +1622,7 @@ parameter_types! {
     // Make sure to stay below MAX_BLOCK_SIZE of substrate consensus of ~4MB
     // The new compressed wasm format is much smaller in size ~ 1MB
     pub const RuntimeUpgradeWasmProposalMaxLength: u32 = DispatchableCallCodeMaxLen::get();
-    pub const FundingRequestProposalMaxAmount: Balance = dollars!(10_000);
+    pub const FundingRequestProposalMaxTotalAmount: Balance = 1_000_000 * currency::DOLLARS; // TODO: adjust
```

Comment:
The value should stay as it was:
```suggestion
pub const FundingRequestProposalMaxAmount: Balance = dollars!(10_000);
```